hackathon_site/dashboard/frontend/src/pages/Inventory/Inventory.js

```diff
@@ -2,13 +2,24 @@ import React from "react";
 // import styles from "./Inventory.module.scss";
```

Comment:
Commented out just because react will throw an unused import warning, but you know we'll need it here eventually?